scripts/demo.sh

```diff
@@ -0,0 +1,4 @@
+# Build demo image and push to dockerhub (you need to login to dockerhub first)
```

Comment:
Does this script need a license header?

Reply:
Let's do this for the whole codebase at once.